ui/src/components/ColorToggleButton.tsx

```diff
@@ -2,7 +2,7 @@ import { MoonIcon, SunIcon } from "@chakra-ui/icons"
 import { IconButton, IconButtonProps, useColorMode } from "@chakra-ui/react"
 import React from "react"
 
-function ColorToggleButton(props: Partial<IconButtonProps>): React.ReactElement {
+function ColorToggleButton(props: Readonly<Partial<IconButtonProps>>): React.ReactElement {
```

Comment:
Why `Readonly`?

Reply:
Sonarlint, this is just a precaution, arguments here should never be modified